src/diffusers/loaders/single_file_utils.py

```diff
@@ -1381,65 +1402,86 @@ def create_scheduler_from_ldm(
     prediction_type=None,
     scheduler_type="ddim",
     model_type=None,
+    is_playground_model=False,
 ):
-    scheduler_config = get_default_scheduler_config()
-    model_type = infer_model_type(original_config, model_type=model_type)
+    if is_playground_model:
+        scheduler_kwargs = {
+            "algorithm_type": "dpmsolver++",
+            "dynamic_thresholding_ratio": 0.995,
+            "euler_at_final": False,
+            "final_sigmas_type": "zero",
+            "lower_order_final": True,
+            "num_train_timesteps": 1000,
+            "prediction_type": "epsilon",
+            "rho": 7.0,
+            "sample_max_value": 1.0,
+            "sigma_data": 0.5,
+            "sigma_max": 80.0,
+            "sigma_min": 0.002,
+            "solver_order": 2,
+            "solver_type": "midpoint",
+            "thresholding": False,
+        }
+        scheduler = EDMDPMSolverMultistepScheduler(**scheduler_kwargs)
+    else:
+        scheduler_config = get_default_scheduler_config()
+        model_type = infer_model_type(original_config, model_type=model_type)
 
-    global_step = checkpoint["global_step"] if "global_step" in checkpoint else None
```

Comment:
In general I want to avoid trying to introduce a lot of nested conditional statements into single file again.